eidas-saml-parser/src/test/java/uk/gov/ida/notification/eidassaml/saml/EidasAuthnRequestValidatorTest.java

```diff
@@ -171,15 +170,11 @@ public void shouldThrowIfAuthnRequestHasProtocolBinding() throws XPathExpression
         eidasAuthnRequestValidator.validate(request);
     }
 
-    @Test
-    public void shouldThrowIfAuthnRequestHasUnsupportedSamlVersion() {
+    @Test(expected = InvalidAuthnRequestException.class)
+    public void shouldThrowIfAuthnRequestHasUnsupportedSamlVersion() throws XPathExpressionException, TransformerException {
         final String expectedMessage = "Bad Authn Request from Connector Node: SAML Version should be " + SAMLVersion.VERSION_20.toString();
 
-        assertThrows(InvalidAuthnRequestException.class, () ->
-                eidasAuthnRequestValidator.validate(eidasAuthnRequestBuilder.withSamlVersion(SAMLVersion.VERSION_10).build()), expectedMessage);
-
-        assertThrows(InvalidAuthnRequestException.class, () ->
-                eidasAuthnRequestValidator.validate(eidasAuthnRequestBuilder.withSamlVersion(SAMLVersion.VERSION_11).build()), expectedMessage);
+        eidasAuthnRequestValidator.validate(eidasAuthnRequestBuilder.withSamlVersion(SAMLVersion.VERSION_10).build());
```

Comment:
https://github.com/alphagov/verify-proxy-node/blob/38c164edd019d82f250f168b2d5a02235f5807c1/eidas-saml-parser/src/test/java/uk/gov/ida/notification/eidassaml/saml/EidasAuthnRequestValidatorTest.java#L175-L177

`expectedMessage` is now not used - it used to validate the exception message. Also two invalid versions of SAML were tested. This can be more elegantly rewritten using the AssertJ exception validation:
` assertThatThrownBy(() -> { eidasAuthnRequestValidator.validate(...VERSION_10...); }).isInstanceOf(InvalidAuthnRequestException.class).hasMessageContaining("expectedMessage");`

And the same for `VERSION_11` - instead of the `expected = InvalidAuthnRequestException.class` flag